### ISSUE TITLE
Doesn't seem to do anything

### ISSUE THREAD

**Issue by elsurudo** [ksylvest/vue-file#1]
If this is a multiple-file-upload solution for vue that and allows direct uploads to AWS, I might be working on the same thing soon :)

**Comment by ksylvest** [comment]
Yup! Haven't had a chance to do much OSS lately and thought this'd be a fun project.

**Comment by elsurudo** [comment]
Nice! What are your requirements? Maybe we can team up. I've got a new project in Rails 5.1.rc1, and decided on Vue for the more complex stuff. I've started a pretty neat implementation of a client-side table with sorting, paging, searching, filtering etc.

And yeah, file uploads are the other place I was thinking of using Vue.

**Comment by elsurudo** [comment]
Well Vue is client-side of course, but the table is meant to interact with the server-side to get its data, depending on the state on the client.
